.changeset/ten-falcons-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Fix flaky tests by making sure @ember/test-waiters run in all environments
```

Comment:
`Modal`, `Flyout` - Fixed flaky tests by running `@ember/test-waiters` in all environments